src/extension/monitor/core/src/main/java/org/geoserver/monitor/MonitorConfig.java

```diff
@@ -228,5 +230,67 @@ Properties props() {
         return props;
     }
 
+    @Override
+    public List<Resource> getFileLocations() throws IOException {
+        List<Resource> configurationFiles = new ArrayList<>();
+        GeoServerResourceLoader loader = GeoServerExtensions.bean(GeoServerResourceLoader.class);
```

Comment:
The loader is already available via construtor injection, why look it up dynamically at runtime?